core/chains/evm/headtracker/head_broadcaster_test.go

```diff
@@ -61,7 +61,7 @@ func TestHeadBroadcaster_Subscribe(t *testing.T) {
 		}).
 		Return(sub, nil)
 	// 2 for initial and 2 for backfill
-	ethClient.On("HeadByNumber", mock.Anything, mock.Anything).Return(testutils.Head(1), nil).Times(4)
+	ethClient.On("HeadByNumber", mock.Anything, mock.Anything).Return(testutils.Head(1), nil)
```

Comment:
Why 4 isn't needed anymore? Do we have a behavioural change? Should we update the comment above?